src/test/java/uk/gov/hmcts/cmc/claimstore/services/ClaimServiceTest.java

```diff
@@ -279,20 +279,6 @@ public void requestMoreTimeToRespondShouldThrowNotFoundExceptionWhenClaimNotFoun
         claimService.requestMoreTimeForResponse(EXTERNAL_ID, AUTHORISATION);
     }
 
-    @Test(expected = MoreTimeRequestedAfterDeadlineException.class)
-    public void requestMoreTimeForResponseThrowsMoreTimeRequestedAfterDeadlineWhenItsTooLateForMoreTimeRequest() {
-
-        LocalDate responseDeadlineInThePast = now()
-            .minusDays(10);
-        Claim claim = createClaimModel(responseDeadlineInThePast, false);
-
-        when(caseRepository.getClaimByExternalId(eq(EXTERNAL_ID), any()))
-            .thenReturn(Optional.of(claim));
-        when(userService.getUser(eq(AUTHORISATION))).thenReturn(USER);
-
-        claimService.requestMoreTimeForResponse(EXTERNAL_ID, AUTHORISATION);
-    }
```

Comment:
Why is this removed? Seems unrelated

Reply:
I don't know how to undo changes to this file so I deleted it because it is deleted in another pr too.